llvm/utils/gn/secondary/llvm/include/llvm/Config/BUILD.gn

```diff
@@ -384,6 +387,12 @@ write_cmake_config("llvm-config") {
   } else {
     values += [ "LLVM_FORCE_ENABLE_STATS=" ]
   }
+}
+
+write_cmake_config("Targets") {
```

Comment:
@nico @aeubanks Alternatively we could just remove this build target; nothing that GN is currently capable of building needs it.